<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>MainWindow</class>
<widget class="QMainWindow" name="MainWindow">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>400</width>
   <height>600</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>Bitmap to SVG Converter</string>
 </property>
 <widget class="QWidget" name="centralwidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="fileGroup">
     <property name="title">
      <string>Input File</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="filePathLabel">
        <property name="text">
         <string>No file selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sizeGroup">
     <property name="title">
      <string>Size Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="pixelSizeRadio">
        <property name="text">
         <string>Pixel Size</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="docSizeRadio">
        <property name="text">
         <string>Document Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mmRadio">
        <property name="text">
         <string>mm</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="inchesRadio">
        <property name="text">
         <string>inches</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="pixelSizeWidget">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="pixelSizeLabel">
           <property name="text">
            <string>Pixel Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="pixelSizeInput">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.001</double>
           </property>
           <property name="maximum">
            <double>1000.000</double>
           </property>
           <property name="value">
            <double>1.000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="documentSizeWidget">
        <layout class="QFormLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="widthLabel">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="docWidthInput">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.001</double>
           </property>
           <property name="maximum">
            <double>10000.000</double>
           </property>
           <property name="value">
            <double>100.000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="heightLabel">
           <property name="text">
            <string>Height:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="docHeightInput">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.001</double>
           </property>
           <property name="maximum">
            <double>10000.000</double>
           </property>
           <property name="value">
            <double>100.000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="strokeGroup">
     <property name="title">
      <string>Stroke Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="strokeWidthLabel">
          <property name="text">
           <string>Stroke Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="strokeWidthInput">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.001</double>
          </property>
          <property name="maximum">
           <double>100.000</double>
          </property>
          <property name="value">
           <double>0.100</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="solidRadio">
        <property name="text">
         <string>Solid</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="zigzagRadio">
        <property name="text">
         <string>Zigzag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="angleWidget">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="angleLabel">
           <property name="text">
            <string>Angle (degrees):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="angleInput">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.0</double>
           </property>
           <property name="maximum">
            <double>360.0</double>
           </property>
           <property name="value">
            <double>45.0</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="optimizeCheckbox">
        <property name="text">
         <string>Optimize paths for plotter</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="optimizeForInkscapeCheckbox">
        <property name="toolTip">
         <string>Reduces file size and improves loading in Inkscape</string>
        </property>
        <property name="text">
         <string>Optimize for Inkscape</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="convertButton">
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</widget>
<resources/>
<connections/>
</ui>